package.json

```diff
@@ -7,53 +7,52 @@
     "android": "react-native run-android",
     "ios": "react-native run-ios",
     "test": "jest",
-    "lint": "eslint ."
+    "lint": "eslint .",
```

Comment:
I think this line causes an error, in the earlier version of nodejs this is not allowed to have a trailing comma in the `package.json` file.